localstack/services/cloudwatch/provider.py

```diff
@@ -226,19 +254,49 @@ def put_metric_alarm(
         context: RequestContext,
         request: PutMetricAlarmInput,
     ) -> None:
+
+        # missing will be the default, when not set (but it will not explicitly be set)
+        if not request.get("TreatMissingData", "missing") in [
+            "breaching",
+            "notBreaching",
+            "ignore",
+            "missing",
+        ]:
+            raise ValidationError(
+                f"The value {request['TreatMissingData']} is not supported for TreatMissingData parameter. Supported values are [breaching, notBreaching, ignore, missing]."
```

Comment:
**nit:** Hard-coding the supported values in this error message might lead to typos further down the line if those values are changed.